### ISSUE TITLE
[icerec] Incremental Release V05-01-03 (Trac #2026)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2018]
I need an incremental release with the latest filterscripts release for L2 offline processing for season 2017 and pass2:

filterscripts -r155712 http://code.icecube.wisc.edu/svn/projects/filterscripts/releases/V17-05-00

Migrated from https://code.icecube.wisc.edu/ticket/2026
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:13:53", 
    "description": "I need an incremental release with the latest filterscripts release for L2 offline processing for season 2017 and pass2:\n\nfilterscripts -r155712 http://code.icecube.wisc.edu/svn/projects/filterscripts/releases/V17-05-00", 
    "reporter": "joertlin", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067233566919", 
    "component": "combo reconstruction", 
    "summary": "[icerec] Incremental Release V05-01-03", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2017-05-18T18:07:52", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170519T13:17:05: 
* kjmeagher changed status from "new" to "closed"
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 155839:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="155839"
> Create incremental release of icerec with support for new filterscripts. fixes #2018
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:13:53: 
* olivas changed milestone from "May 2017" to ""
* olivas commented:

> Milestone May 2017 deleted
